OutOfSchool/OutOfSchool.WebApi/Services/SubordinationStructure/InstitutionHierarchyService.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using OutOfSchool.Services.Models.SubordinationStructure;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.WebApi.Common;
+using OutOfSchool.WebApi.Models.SubordinationStructure;
+
+namespace OutOfSchool.WebApi.Services.SubordinationStructure
+{
+    public class InstitutionHierarchyService : IInstitutionHierarchyService
```

Comment:
Is there any possibility to cover this class functionality with unit-test?

Reply:
Done